cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -35,7 +35,7 @@
     {% if video_url.find( 'autoplay=' ) == -1 -%}
         {% set video_url = _buildParam( video_url, 'autoplay=1' )  %}
     {% endif %}
-    {% set video_id =  video_url.split('?')[0].split('/') | last %}
+    {% set video_id = video_url.split('?')[0].split('/') | last if video_id == '' else video_id %}
```

Comment:
__Out of scope:__ We should rethink including fields for both the ID and URL. They're essentially the same information, and two inputs increases the risk that one could be off. I'd suggest dropping the URL field and building the URL string based on the ID. There's less chance a bunch of cruft ending up in the final URL string.

Reply:
That's what I was referring to with my note in the PR. The issue is that the different Wagtail modules that call the video player macro (FCM, Video Player, and Event) aren't consistent in the data that they request. Ideally, I agree, we'd standardize on just asking for the video ID.

Reply:
Ahhh, I didn't realize that was the difference. Any idea if this has been logged with the team or where the priority may lie?